kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -2414,6 +2414,16 @@ messages:
 
       AddPacket(4,iFlags);
 
+      % Send special type if special flag is set
```

Comment:
Maybe this would be better done by a new function (on Item?  Not sure) that returns the "special type".  So then this would just be

```AddPacket(4, Send(what, @GetSpecialType)); ```
(with a better name perhaps)

Come to think of it, if you're always adding 4 bytes to every message anyway, what is the purpose of ITEM_SPECIAL_YES?  Could the client simply test if the special type is 0?

Reply:
Agreed.  I have created a new function in `item.kod` as suggested and will refine it.

I have also eliminated the `ITEM_SPECIAL_YES` flag completely, so there's still that bit available for something else one day.